profile/README.md

```diff
@@ -1,38 +1,48 @@
+**Diagram**. Frawework illustrates the main blocks for acquisition, base, computer vision, interface, and recorder, along with their corresponding ROS 2 packages. Highlight the green blocks as already implemented and the yellow ones as in progress. See also logo [here](../docs/figs/sense-base-logo.svg)
 
-## :school_satchel: Getting started
-* :computer: [Setting up ROS2 with docker container](https://github.com/sense-base/base/tree/main/docs/docker)
+## 1. :robot: SENSE-BASE ROS2-based packages
+* [sense_base](https://github.com/sense-base/base) Main repo to manage packages.
+* [sense_eeg](https://github.com/sense-base/sense_eeg) Mock EEG publisher
+* [sense_msgs](https://github.com/sense-base/sense_msgs) ROS2 messages 
+* [sense_bridge](https://github.com/sense-base/sense_bridge) Manage multi-dimensional arrays
+* [sense_pose](https://github.com/sense-base/sense_pose) Pose estimation tools
+* [sense_template](https://github.com/sense-base/sense_template) Template for ROS2 packages
 
-* :octocat: Clone repos under `sense-base` path and refer to the [CONTRIBUTING](CONTRIBUTING.md) guideline for detailed instructions on contributing to the project.
+## 2. :school_satchel: Getting started by install requirements and dependencies
+* Generate your SSH keys as suggested [here](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent).
+* Setup you commit signature verification as shown [here](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification#ssh-commit-signature-verification).
+* :octocat: Clone base repositories under `sense-base` path.
+We suggest cloning repos under sense-base path as for example `base`, `sense_eeg` and `sense_msgs`.
 ```
-# For example:
+mkdir sense-base && cd sense-base
+git clone git@github.com:sense-base/base.git
+cd base && mkdir -p workspace/src && cd workspace/src
 git clone git@github.com:sense-base/sense_eeg.git
+git clone git@github.com:sense-base/sense_msgs.git
+git clone git@github.com:sense-base/sense_bridge.git
 ```
+* :computer: [Setting up ROS2 with docker container (including docker dependencies)](https://github.com/sense-base/base/tree/main/docs/docker)
 
-* :nut_and_bolt: Run and debug. Open a terminal into the loaded container in VSCode using the dev containers extension, and run
-```
-colcon build --symlink-install
-source install/setup.bash
-ros2 launch eeg_publisher mock_publisher_launch.py
-```
+## 3. :nut_and_bolt: Install local virtual environments
+### Setup environment
+Install Python Environment (3.10 or higher recommended):
 
-On a different terminal, run
+#### Using uv
 ```
-source install/setup.bash
-ros2 topic list
-ros2 topic echo /eeg/raw
-ros2 bag record /eeg/raw
-ros2 bag info <bag_file_name> (e.g., `rosbag2_data_time`)
-ros2 bag play <bag_file_name>
-rqt_graph
+#cd package name
+cd workspace/src/sense_msgs/
+uv venv --python 3.10
```

Comment:
How does this play with the venv declared in the Dockerfile?

Reply:
Let's address it here https://github.com/sense-base/base/issues/33